independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/matching/RegexMatchTest.java

```diff
@@ -41,6 +41,10 @@ public void testLiteralInRegex() {
                 .then()
                 .statusCode(200)
                 .body(equalTo("plain:abb/foo/alongpathtotriggerbug"));
+        RestAssured.get("/regex/first space/foo/second space")
+                .then()
+                .statusCode(200)
+                .body(equalTo("plain:first space/foo/second space"));
```

Comment:
Is this the only new test we need?

Reply:
:smile: For the new issue yes. Paths were not decoded when using a custom regex like the one in `RegexResource`. This test failed without the fix.

Working on this tcks folder tests failed initially. I had not changed all the calls, that's why I changed the signature, to be sure I was not missing anything. So I hope those tests are testing the common functionality. I can add more tests if you see something missing...

Reply:
Cool, let's see what CI says